app/code/Magento/Checkout/view/frontend/templates/success.phtml

```diff
@@ -5,22 +5,27 @@
  */
 
 ?>
-<?php /** @var $block \Magento\Checkout\Block\Onepage\Success */ ?>
+<?php
```

Comment:
Please remove here not needed closing and opening PHP tags.

Reply:
fixed